.eslintrc.json

```diff
@@ -42,6 +42,7 @@
     "import/no-unresolved": 0,
     "import/export": 0,
     "func-names": 0,
-    "semi": [1, "always"] // 1 is for warning
+    "semi": [1, "always"], // 1 is for warning
+    "@next/next/no-img-element": "off"
```

Comment:
This was noisy. We don't really need the next/image for optimization but the blur and lazy load is nice. Could we move to using next/image everywhere?

Reply:
Must be on the same wavelength because I just did that today too 😆 
https://github.com/SU-SWS/ood-giving-campaign/pull/128/commits/93f028a56b4cf1a54c6e74da8357a37171439d92

I tried using the next/image but I don't like how that was a delay, and it doubles up with the Storyblok image services. I'm using the browser native lazy load and that seems to be sufficient (esp now we're using webp and even the large images are small in file size).

Reply:
Check out these small image file sizes 😄 

![Screenshot 2023-10-04 at 12 25 02 AM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/09c63a1f-cb48-49d1-93ce-936b2d8df571)

![Screenshot 2023-10-04 at 12 24 40 AM](https://github.com/SU-SWS/ood-giving-campaign/assets/42749717/f0fbcaee-db73-4470-aa90-b9d30107782c)